project/Release.scala

```diff
@@ -0,0 +1,59 @@
+import sbt.Keys._
+import sbt.{Command, _}
+
+import scala.sys.process
+
+object Release {
+
+  // Main release commands
+  private val stryker4sPublish = "stryker4sPublish"
+  private val stryker4sPublishSigned = "stryker4sPublishSigned"
+  // Helper command names
+  private val stryker4sMvnSetup = "stryker4sMvnSetup"
+  private val stryker4sMvnPackage = "stryker4sMvnPackage"
+  private val stryker4sMvnDeploy = "stryker4sMvnDeploy"
+  private val publishM2 = "stryker4s-core/publishM2"
+  private val `+publish` = "+publish"
+  private val `+publishSigned` = "+publishSigned"
+
+  lazy val releaseCommands: Setting[Seq[Command]] = commands ++= {
+    val versionNumber = version.value
+    Seq(
+      // Called by sbt-ci-release
+      Command.command(stryker4sPublish)(stryker4sMvnSetup :: `+publish` :: stryker4sMvnDeploy :: _),
+      Command.command(stryker4sPublishSigned)(stryker4sMvnSetup :: `+publishSigned` :: stryker4sMvnDeploy :: _),
+      // Called by stryker4sPublish(signed)
+      Command.command(stryker4sMvnSetup)(publishM2 :: stryker4sMvnPackage :: setVersion(versionNumber) :: _),
+      Command.command(stryker4sMvnPackage)(mvnPackage(versionNumber, baseDirectory.value)),
+      Command.command(stryker4sMvnDeploy)(mvnDeploy(baseDirectory.value))
+    )
+  }
+
+  private def mvnPackage(version: String, baseDir: File): State => State = handleProcessResult(
```

Comment:
True. I did this as a first step so that you wouldn't run into the scenario of publishing the sbt project, and then finding out the maven project didn't compile or something. I've now added a test stage, so this is a bit redundant. I'll remove this and set the version in mvnDeploy